lib/babe/verify_test.go

```diff
@@ -648,13 +647,14 @@ func Test_verifier_verifyAuthorshipRight(t *testing.T) {
 			name:     "valid digest items, getAuthorityIndex error",
 			verifier: *babeVerifier5,
 			header:   header7,
-			expErr:   errors.New("failed to get authority index no digest provided"),
+			expErr: fmt.Errorf("could not verify block equivocation: failed to get authority index for block %s: for block hash %s: %w", //nolint:lll
+				h, types.NewEmptyHeader().Hash(), errNoDigest),
```

Comment:
removing the `//nolint` + expecting constant string since there is no need to have a dynamic expectation (remove complexity/debugging hassle)

```suggestion
			expErr: fmt.Errorf("could not verify block equivocation: " +
				"failed to get authority index for block " +
				"0x0100000000000000000000000000000000000000000000000000000000000000: " +
				"for block hash 0xdcdd89927d8a348e00257e1ecc8617f45edb5118efff3ea2f9961b2ad9b7690a: " +
				"no digest provided"),
```